scripts/installation/install-openkat.yml

```diff
@@ -0,0 +1,588 @@
+    get_url:
+      url: "https://github.com/minvws/nl-kat-coordination/releases/download/v{{ katrelease }}/kat-debian12-{{ katrelease }}.tar.gz"
+      dest: /tmp
+      mode: '0440'
+
+  - name: "Download release xtdb {{ xtdbrelease }} to our Debian 12 server"
+    get_url:
+      url: "https://github.com/dekkers/xtdb-http-multinode/releases/download/v{{ xtdbrelease }}/xtdb-http-multinode_{{ xtdbrelease }}_all.deb"
+      dest: /tmp
+      mode: '0440'
+
+  # After downloading they can be installed as follows:
+  # tar zvxf kat-*.tar.gz
+  - name: Unpack tar OpenKAT file
+    unarchive:
+      src: "/tmp/kat-debian12-{{ katrelease }}.tar.gz"
+      remote_src: yes
+      dest: /tmp
+
+  # sudo apt install --no-install-recommends ./kat-*_amd64.deb ./xtdb-http-multinode_*_all.deb
+  - name: Install all needed packages
+    apt:
+      deb: "/tmp/{{ item }}"
+      install_recommends: false
+    loop:
+      - "kat-boefjes_{{ katrelease }}_amd64.deb"
+      - "kat-bytes_{{ katrelease }}_amd64.deb"
+      - "kat-keiko_{{ katrelease }}_amd64.deb"
+      - "kat-mula_{{ katrelease }}_amd64.deb"
+      - "kat-octopoes_{{ katrelease }}_amd64.deb"
+      - "kat-rocky_{{ katrelease }}_amd64.deb"
+      - "xtdb-http-multinode_{{ xtdbrelease }}_all.deb"
+
+  # Set up the databases
+  # OpenKAT needs three databases for its components. One for rocky, KAT-alogus and bytes. The following steps will guide you through the creation of these databases.
+  # If you will be running the database on the same machine as KAT, you can install Postgres:
+  # sudo apt install postgresql
+  - name: Install other needed packages
+    apt:
+      name: "{{ item }}"
+      state: present
+    loop:
+      - python3-psycopg2
+      - python3-pexpect
+      - postgresql
+      - postgresql-client
+      - rabbitmq-server
+      - sudo
+
+  # Rocky DB
+  # Generate a secure password for the Rocky database user, as an example we’ll use /dev/urandom:
+  # echo $(tr -dc A-Za-z0-9 < /dev/urandom | head -c 20)
+  - name: Generate random password for Rocky DB
+    set_fact:
+      # Turns out that special characters create problems.
+      #var: "{{ lookup('community.general.random_string', length=20, min_numeric=3, min_upper=3, min_special=3) }}"
+      rocky_db_password: "{{ lookup('password', '/dev/null', length=20, chars=['ascii_letters'] ) }}"
+
+  - name: Show password for debugging purposes (TODO comment these lines)
+    debug:
+      var: rocky_db_password
+
+  # To configure rocky to use this password, open /etc/kat/rocky.conf and fill in this password for ROCKY_DB_PASSWORD.
+  - name: Store password in /etc/kat/rocky.conf
+    lineinfile:
+      path: /etc/kat/rocky.conf
+      regexp: "ROCKY_DB_PASSWORD=.*"
+      line:  "ROCKY_DB_PASSWORD={{ rocky_db_password }}"
+
+  # sudo -u postgres createuser rocky -P
+  - name: Create Postgresql user rocky and grant privs
+    become: true
+    become_user: postgres
+    become_method: sudo
```

Comment:
become_method kan beter weggelaten worden, zodat dit eventueel in de user's config gezet kan worden of kan overruled worden daar.

Reply:
Ja, maar dat kan prima zonder 'become_method' ... become_user: postgres en become: true zijn hier voldoende voor... en laten mensen andere alternatieven (naast sudo) gebruiken in hun lokale config.
Dus alleen alle 'become_method:' regels verwijderen zou de situatie verbeteren zonder enige impact.

Reply:
Ah, juist. Ok.